src/main/java/core/basesyntax/impl/ReadingCsvFileImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.CantWorkWithThisFileException;
```

Comment:
The exception 'CantWorkWithThisFileException' is a custom exception, but it's not clear what conditions it represents. It's better to use a more specific naming convention that reflects the exceptional scenario, like 'FileReadingException'.